admin/webUI/app/components/invite/inviteController.js

```diff
@@ -299,6 +312,121 @@
     }, function(err) {
       console.log(err);
     });
+
+    function showModal(candidateID, email) {
+
+      console.log(email)
+      candidatesVm.candidate_email = email;
+      dialog.showModal();
+      dialog.querySelector('.submit').addEventListener('click', function() {
+        candidatesVm.cancel(candidateID);
+        dialog.close();
+      })
+    }
+
+    function showCancelModal(candidateID, email) {
+      console.log(email)
+      candidatesVm.candidate_email = email;
+      // dialog.showModal();
+      mainVm.openModal({
+        template: $(".canel-invite-template").html(),
+        isString: true,
+        showYes: true,
+        class: "cancel-invite-modal"
+      });
+      dialog.querySelector('.submit').addEventListener('click', function() {
+        candidatesVm.cancel(candidateID);
+        dialog.close();
+      })
+    }
+
+    angular.element(document).ready(function() {
+      var dialog = document.querySelector('dialog');
+
+      dialog.querySelector('.close').addEventListener('click', function() {
+        dialog.close();
+      });
+    });
+
+    function expires(validity) {
+      var numDays = moment(validity).diff(moment(), 'days')
+      if (numDays == 0) {
```

Comment:
Moment.js is awesome library, saves time (not pun intended) and easy to use. We should use it only if we will be dealing with time/date heavily. Else we can work around in javascript.  We should try to avoid using external libraries because it will add unnecessary load time by adding 68-70kb of file request for doing one task only. 
Below code will help you to find difference between "validity date" and "today" in days. 

Can decide about this. Moment.js is (time)saviour, has useful methods.
```js
var validity_date = new Date(validity)
var today = new Date();
var diff =  (validity_date - today)/(1000*60*60*24)
noOfDays = Math.round(diff)
```